docs/reference/ml/df-analytics/apis/delete-dfanalytics.asciidoc

```diff
@@ -23,6 +23,7 @@ If the {es} {security-features} are enabled, you must have the following
 built-in roles or equivalent privileges:
 
 * `machine_learning_admin`
+* cluster: `manage_ml` or `manage`
```

Comment:
Should the `machine_learning_admin` entry above this line be in the same line as `cluster`?